app/assets/stylesheets/modules/homepage.scss

```diff
@@ -39,3 +39,83 @@ body.home-page {
     background-color: #008080;
   }
 }
+
+//SEARCH FORM
+.box{
+  width: 100%;
+  height: 50px;
+}
+
+.container-4{
+  overflow: hidden;
+  width: 100%;
+  vertical-align: middle;
+  white-space: nowrap;
+}
+
+.container-4 #location{
+  width: 100%;
+  height: 50px;
+  background: #ffffff;
+  border: none;
+  font-size: 10pt;
+  float: left;
+  color: #000;
+  padding-left: 15px;
+  -webkit-border-radius: 5px;
+  -moz-border-radius: 5px;
+  border-radius: 5px;
+}
+
+.container-4 #location::-webkit-input-placeholder {
+   color: #65737e;
+}
+
+.container-4 #location:-moz-placeholder { /* Firefox 18- */
+   color: #65737e;
+}
+
+.container-4 #location::-moz-placeholder {  /* Firefox 19+ */
+   color: #65737e;
+}
+
+.container-4 #location:-ms-input-placeholder {
+   color: #65737e;
```

Comment:
Color literals like `#65737e` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 3 spaces